sdk/privatedns/Azure.ResourceManager.PrivateDns/tests/Scenario/PrivateDnsTests.cs

```diff
@@ -174,6 +189,47 @@ public async Task AddRemoveTag(bool? useTagResource)
             Assert.AreEqual(0, privateZone.Data.Tags.Count);
         }
 
+        [RecordedTest]
+        public async Task AddVnetLinkWithResolutionPolicy()
+        {
+            string privateZoneName = "privatelink.sql.azuresynapse.net";
+            string vnetName = "samplevnet";
+            string vnetLinkName = $"{Recording.GenerateAssetName("samplelink")}";
+            var privateZone = await CreatePrivateZone(_resourceGroup, privateZoneName);
+            var vnet = await CreateVirtualNetwork(_resourceGroup, vnetName);
+
+            // Add NxDomainRedirect resolutionPolicy
+            var virtualNetworkLinkData = new VirtualNetworkLinkData("global")
+            {
+                VirtualNetwork = new WritableSubResource { Id = vnet.Id },
+                RegistrationEnabled = false,
+                ResolutionPolicy = ResolutionPolicy.NxDomainRedirect,
+            };
+
+            await CreateOrUpdateVirtualNetworkLink(privateZone, vnetLinkName, virtualNetworkLinkData);
+
+            if (TestEnvironment.Mode == RecordedTestMode.Record)
```

Comment:
The above operation is a long running operation, and you invoke it with `WaitUntil.Completed`. So, here we should not wait another 30 seconds, right?

Reply:
Yes you're right, that was unnecessary. Removed it in the latest commit.